File: ComputerSeekho/src/main/java/com/example/entities/News.java
package com.example.entities;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "news")
public class News {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "News_id")
    private int newsId;

    @Column(name = "News_url", length = 255, nullable = false)
    private String newsUrl;

    @Column(name = "News_title", length = 255, nullable = false)
    private String newsTitle;

    @Column(name = "News_description", length = 255, nullable = false)
    private String newsDescription;

}

File: ComputerSeekho.Net/ComputerSeekho.Net/Models/News.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ComputerSeekho.Net.Models;

[Table("news")]
public class News
{
    [Key]
    [Column("News_id")]
    public int NewsId { get; set; }

    [Column("News_title")]
    public string? NewsTitle { get; set; }

    [Column("News_description")]
    public string? NewsDescription { get; set; }

    [Column("News_url")]
    public string? NewsUrl { get; set; }
}
